src/Environment.js

```diff
@@ -0,0 +1,100 @@
+/* @flow */
+
+import invariant from 'invariant';
+import Relay from 'react-relay';
+import RelayNetworkLayer from 'react-relay/lib/RelayNetworkLayer';
+import RelayStoreData from 'react-relay/lib/RelayStoreData';
+
+import createSubscriptionQuery from './createSubscriptionQuery';
+import type Subscription from './Subscription';
+import SubscriptionRequest from './SubscriptionRequest';
+import type {
+  SubscriptionDisposable,
+  SubscriptionObservable,
+  SubscriptionObserver,
+} from './types';
+import updateStoreData from './updateStoreData';
+
+// Override a few Relay classes to use our own network layer proxy that
+// supports sendSubscription.
+
+class NetworkLayer extends RelayNetworkLayer {
+  sendSubscription(
```

Comment:
I wonder if a better API might be `(SubscriptionRequest, Observer) => Disposable`.